be/src/vec/exec/volap_scan_node.cpp

```diff
@@ -207,46 +162,37 @@ void VOlapScanNode::scanner_thread(VOlapScanner* scanner) {
     // scan, if this exceed threshold, we yield this thread.
     int64_t raw_rows_read = scanner->raw_rows_read();
     int64_t raw_rows_threshold = raw_rows_read + config::doris_scanner_row_num;
-    while (!eos && raw_rows_read < raw_rows_threshold) {
+    bool not_miss_block = true;
```

Comment:
i donot know what's the meaning of not_miss_block, rethink a more meanful name?

Reply:
ok